cpu/stm32/periph/backup_ram.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2021 Otto-von-Guericke Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_stm32
+ * @{
+ *
+ * @file
+ * @brief       Implementation of backup SRAM capabilities for stm32 MCUs
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ * @}
+ */
+
+#include "periph_cpu.h"
+#include "stmclk.h"
+
+#ifndef BKPRAM_CONFIG
+/**
+ * @brief   Configuration to retain backup RAM content during standby mode
+ */
+#if defined(CPU_FAM_STM32F7)
+#define BKPRAM_CONFIG       (PWR_CSR1_BRE | PWR_CSR1_EIWUP)
+#else
+#define BKPRAM_CONFIG       (PWR_CSR_BRE)
+#endif
+#endif
+
+#ifndef BKPRAM_READY
```

Comment:
When would this be defined earlier?

Reply:
I adopted the style of the `periph/pm` driver. Having to `#define BKPRAM_CONFIG` and `#define BKPRAM_READY` in `periph_cpu.h` would rather mean that something is wrong in the logic of `backup_ram.c`. So I would actually drop both: `#ifndef BKPRAM_CONFIG` and `#ifndef BKPRAM_READY`.